public/app/api/legacy/user/api.ts

```diff
@@ -0,0 +1,25 @@
+import { createApi } from '@reduxjs/toolkit/query/react';
+
+import { createBaseQuery } from 'app/api/createBaseQuery';
+
+export const legacyUserAPI = createApi({
+  reducerPath: 'legacyUserAPI',
+  baseQuery: createBaseQuery({ baseURL: '/api' }),
+  tagTypes: ['dashboardStars'],
+  endpoints: (build) => ({
+    getStars: build.query<string[], void>({
+      query: () => ({ url: '/user/stars' }),
+      providesTags: ['dashboardStars'],
+    }),
+    starDashboard: build.mutation<void, { id: string }>({
+      query: ({ id }) => ({ url: `/user/stars/dashboard/uid/${id}`, method: 'POST' }),
+      invalidatesTags: ['dashboardStars'],
+    }),
+    unstarDashboard: build.mutation<void, { id: string }>({
+      query: ({ id }) => ({ url: `/user/stars/dashboard/uid/${id}`, method: 'DELETE' }),
+      invalidatesTags: ['dashboardStars'],
+    }),
+  }),
+});
+
+export const { useGetStarsQuery, useStarDashboardMutation, useUnstarDashboardMutation } = legacyUserAPI;
```

Comment:
Have you done this manually? Don't see any changes in the generate-rrk-apis file 🤔

Reply:
I did do it manually as I didn't realise it was actually defined in the OpenAPI file 😌 I'll move it over either in API clients PR or here, depending on what goes in first?